### PR TITLE
docs: Add training doc

### PR DESCRIPTION
**Pull request by alexec**
image

### CODE REVIEW

**Review by terrytangyuan — APPROVED** [review]
Looks great! Are you planning to migrate the course assets to this repo too?

File: docs/training.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Training
+
+## Hands-On
+
+We're created a Katacoda course featuring beginner and intermediate lessons. These allow to you try out Argo Workflows
```
Comment (dinever) [inline docs/training.md:5]:
```suggestion
We've created a Katacoda course featuring beginner and intermediate lessons. These allow to you try out Argo Workflows
```

File: docs/training.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+in your web browser without needing to install anything on your computer. Each lessons starts up a Kubernetes cluster
```
Comment (dinever) [inline docs/training.md:6]:
```suggestion
in your web browser without needing to install anything on your computer. Each lesson starts up a Kubernetes cluster
```

**Review by dinever — APPROVED** [review]
Looks great! Just pointing out some typos

**Comment by alexec** [comment]
> Looks great! Are you planning to migrate the course assets to this repo too?

No. Changing this repo requires PRs and each needs a detailed review. I want to avoid that for Katacoda lessons.

**Review by sarabala1979 — APPROVED** [review]

**Pull request merged**